app/models/task_list/section.rb

```diff
@@ -1,12 +1,17 @@
 class TaskList::Section
   attr_accessor :identifier, :tasks
 
-  def initialize(identifier:, tasks:)
+  def initialize(identifier:, tasks:, locales_path:)
+    @locales_path = locales_path
     @identifier = identifier
     @tasks = tasks
   end
 
   def title
```

Comment:
I think this `title` method can go now, along the `title` method in `TaskList::Task` and most of the `task_list.en.yml`, they were part of the previous implementation.

Reply:
I thought I remembered trying that... I'll have  a look.

Reply:
The section one looks like it can.

This helper task_status_tag(task) looks like it is used for both new and old as well so needs work.

Reply:
SImplest thing to do right now is to keep title on TaskList::Task and remove it later?

Reply:
As you proposed, we've changed the `task_status_tag` method to take the appropriate id as a temporary solution.